app/controllers/events/meetsReqsForEvent.py

```diff
@@ -0,0 +1,25 @@
+from app.models.user import User
+from app.models.event import Event
+from app.models.eventParticipant import EventParticipant
+from app.models.programBan import ProgramBan
+
+
+def isEligibleForProgram(event, user):

```

Comment:
It doesn't make sense for this function to take an event as an argument, does it?

The function name indicates that you're checking if a user is eligible for a program, so the arguments should be `user` and `program`. We will need to use this function outside of the context of an event (like if we're showing on a user's profile that they are barred from a program).

Reply:
We changed the parameter to `programId` and `user`